### PR TITLE
Set up CI

### PR DESCRIPTION
**Pull request by dmitryduev**
This PR decreases the entropy of the Universe.

### CODE REVIEW

File: .github/workflows/lint.yaml [hunk 1]
```diff
@@ -1,16 +1,11 @@
...
 on:
-  # Triggers the workflow on push or pull request events but only for the master branch
   push:
     branches: [ master ]
   pull_request:
-    # branches: [ master ]
-  # Allows you to run this workflow manually from the Actions tab
-  workflow_dispatch:
-# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+
```
Comment (acrellin) [inline .github/workflows/lint.yaml:7]: No linting on PRs?

**Review by acrellin — APPROVED** [review]

**Comment by stefanv** [comment]
Good stuff, Dima!  I would also enable linting for PRs, otherwise best to get this in ASAP.

**Comment by dmitryduev** [comment]
@acrellin @stefanv I'm confused how `on.pull_request` actually works. Looks like it's running even though the value is null, as long as the key is there, see e.g. https://github.com/dmitryduev/kowalski/pull/49

**Comment by dmitryduev** [comment]
Yea, see here as well:
https://github.com/skyportal/skyportal/blob/master/.github/workflows/test.yaml#L7

**Pull request merged**